### PR TITLE
Fix Persons' Names on BEOL landing Page

### PR DESCRIPTION
**Pull request by tobiasschweizer**

### CODE REVIEW

File: src/app/view/templates/beol/beol.component.ts [hunk 1]
```diff
@@ -116,23 +116,23 @@ export class BeolComponent implements OnInit {
...
+        const Johann_Wendel_Bilfinger = new Correspondent("Johann Wendelin Bilfinger", "(DE-588)104171057");
 
         const Pierre_Bouguer = new Correspondent("Pierre Bouguer", "(DE-588)117622974");
 
-        const Marc_Michel_Bousquet = new Correspondent("Marc Michel Bousquet", "(DE-588)136952968");
+        const Marc_Michel_Bousquet = new Correspondent("Marc-Michel Bousquet", "(DE-588)136952968");
 
         const William_Burnet = new Correspondent("William Burnet", "(DE-588)104185643");
 
-        const Johann_Buxtorf = new Correspondent("Johann Buxtorf", "(DE-588)12068943X");
```
Comment (SepidehAlassi) [inline src/app/view/templates/beol/beol.component.ts]: The name was correct as it was before  (DE-588)12068943X corresponds to Johann Buxtorf both in name index and in DNB
https://portal.dnb.de/opac.htm?method=showFullRecord&currentResultId=12068943X%26any&currentPosition=3

File: src/app/view/templates/beol/beol.component.ts [hunk 2]
```diff
@@ -154,15 +154,15 @@ export class BeolComponent implements OnInit {
 
         const Du_Fresne = new Correspondent("Du Fresne", "(DE-588)1142274934");
 
-        const Amedee_Francois_Frezier = new Correspondent("Amedee Francois Frezier", "(DE-588)117537373");
+        const Amedee_Francois_Frezier = new Correspondent("Amedee François Frezier", "(DE-588)117537373");
```
Comment (SepidehAlassi) [inline src/app/view/templates/beol/beol.component.ts]: actually the correct name should be "Amédée François Frézier" according to name index and DNB
https://portal.dnb.de/opac.htm?query=117537373&method=simpleSearch&cqlMode=true

File: src/app/view/templates/beol/beol.component.ts [hunk 3]
```diff
@@ -154,15 +154,15 @@ export class BeolComponent implements OnInit {
...
 
-        const S_J_Vitus_Friesl = new Correspondent("S J Friesl", "(DE-588)10012786X");
+        const S_J_Vitus_Friesl = new Correspondent("S J Vitus Friesl", "(DE-588)10012786X");
```
Comment (SepidehAlassi) [inline src/app/view/templates/beol/beol.component.ts]: actually the first name does not have S J
name corresponding to this gnd number in both  nameindex and dnb is: Vitus, Friesl
https://portal.dnb.de/opac.htm?query=10012786X&method=simpleSearch&cqlMode=true


--

File: src/app/view/templates/beol/beol.component.ts [hunk 4]
```diff
@@ -243,7 +243,7 @@ export class BeolComponent implements OnInit {
 
         const Johann_Daniel_Schoepflin = new Correspondent("Johann Daniel Schoepflin", "(DE-588)118610260");
 
-        const Willem_Jacob_SGravesande = new Correspondent("Willem Jacob SGravesande", "(DE-588)118939297");
+        const Willem_Jacob_SGravesande = new Correspondent("Willem Jacob sGravesande", "(DE-588)118939297");
```
Comment (SepidehAlassi) [inline src/app/view/templates/beol/beol.component.ts]: name should be Willem Jacob Gravesand

File: src/app/view/templates/beol/beol.component.ts (hunk 1 above)
Comment (tobiasschweizer) [reply src/app/view/templates/beol/beol.component.ts]: done in https://github.com/dhlab-basel/Salsah/pull/206/commits/a1e7e6768121de38743b51670be2e1168d3b6395

File: src/app/view/templates/beol/beol.component.ts (hunk 2 above)
Comment (tobiasschweizer) [reply src/app/view/templates/beol/beol.component.ts]: done in https://github.com/dhlab-basel/Salsah/pull/206/commits/a1e7e6768121de38743b51670be2e1168d3b6395

File: src/app/view/templates/beol/beol.component.ts (hunk 3 above)
Comment (tobiasschweizer) [reply src/app/view/templates/beol/beol.component.ts]: done in https://github.com/dhlab-basel/Salsah/pull/206/commits/a1e7e6768121de38743b51670be2e1168d3b6395

File: src/app/view/templates/beol/beol.component.ts (hunk 4 above)
Comment (tobiasschweizer) [reply src/app/view/templates/beol/beol.component.ts]: sGravesand is a variant: http://d-nb.info/gnd/118939297

in BEBB, the person appears under "S"

**Comment by tobiasschweizer** [comment]
@SepidehAlassi Ok now?

File: src/app/view/templates/beol/beol.component.ts (hunk 4 above)
Comment (SepidehAlassi) [reply src/app/view/templates/beol/beol.component.ts]: let's leave it like this then :-)

**Pull request merged**